packages/dropdown/src/__snapshots__/Dropdown.test.js.snap

```diff
@@ -739,8 +739,8 @@ exports[`Dropdown renders with default value 1`] = `
       </span>
     </div>
     <div
-      aria-selected={false}
-      className="hig__dropdown-option"
+      aria-selected={true}
+      className="hig__dropdown-option hig__dropdown-option--selected"
```

Comment:
Unclear why I needed to update the dropdown snapshots. I don't believe I touched this component.

Reply:
Fixes made in Downshift. `development` should have the updated snapshots.